content/docs/tasks/security/authn-policy/index.md

```diff
@@ -98,12 +93,27 @@ spec:
 EOF
 {{< /text >}}
 
+This policy specifies that all servers in the mesh will only accept encrypted requests using TLS. As you can see, this authentication policy has the kind: `MeshPolicy`. The name of the policy must be `default`, and it contains no `targets` specification (as it is intended to apply on all services in the mesh).
+
+At this point, only the receiving side is configured to use mutual TLS. If you run the curl command between *Istio services* (i.e those with Envoy sidecar), all will fail with 503 error code as the client side still using plain-text.
+
+{{< text bash >}}
+$ for from in "foo" "bar"; do for to in "foo" "bar"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl http://httpbin.${to}:8000/ip -s -o /dev/null -w "sleep.${from} to httpbin.${to}: %{http_code}\n"; done; done
+sleep.foo to httpbin.foo: 503
+sleep.foo to httpbin.bar: 503
+sleep.bar to httpbin.foo: 503
+sleep.bar to httpbin.bar: 503
+{{< /text >}}
+
+To configure client side, you need to set [destination rules](/docs/concepts/traffic-management/#rule-destinations) to use mutual TLS. It's possible to use multiple destination rules, one for each applicable service (or namespace). However, it's more convenient to use a rule with wildcard `*` to match all services so that it is on par with the mesh-wide authentication policy.
```

Comment:
configure client -> configure the client
wildcard -> wildcards

Reply:
Done